xCAT name consistency

gurevichmark opened the pull request:
### The PR is to fix issue _#5662_

### The modification include

_Make "Extreme Cluster/Cloud Administration Tool" consistent in xCAT documentation_

### The UT result
```
[gurevich@c910loginx01 xcat-core]$ git grep -i 'c.* administration t.*' | more
docs/source/index.rst:Extreme Cluster/Cloud Administration Toolkit
perl-xCAT/README:xCAT - Extreme Cluster/Cloud Administration Toolkit
xCAT-OpenStack-baremetal/xpod2man:B<xcat> - Extreme Cluster/Cloud Administration Toolkit.
xCAT-OpenStack-baremetal/xpod2man:Extreme Cluster/Cloud Administration Toolkit (xCAT). xCAT is a scalable distrib
uted computing management
xCAT-OpenStack-ironic/ironic_baremetal/README.rst:xCAT is a Extreme Cluster/Cloud Administration Toolkit. We can
use xcat
xCAT-SoftLayer/xpod2man:B<xcat> - Extreme Cluster/Cloud Administration Toolkit.
xCAT-SoftLayer/xpod2man:Extreme Cluster/Cloud Administration Toolkit (xCAT). xCAT is a scalable distributed compu
ting management
xCAT-UI/debian/control: Provides a browser-based interface for xCAT (Extreme Cluster/Cloud Administration Toolkit
).
xCAT-UI/lib/ui.php:                <tr><td align=right><img src="images/logo.png" width="50" height="35"></img></
td><td colspan=4 style="font-size: 18px;">Extreme Cluster/Cloud Administration Toolkit</td></tr>
xCAT-UI/service.php:                <tr><td align=right><img src="images/logo.png" width="50" height="35"></img><
/td><td colspan=4 style="font-size: 18px;">Extreme Cluster/Cloud Administration Toolkit</td></tr>
xCAT-UI/xCAT-UI.spec:Provides a browser-based interface for xCAT (Extreme Cluster/Cloud Administration Toolkit).
xCAT-client/xpod2man:B<xcat> - Extreme Cluster/Cloud Administration Toolkit.
xCAT-client/xpod2man:Extreme Cluster/Cloud Administration Toolkit (xCAT). xCAT is a scalable distributed computin
g management
xCAT-confluent/xpod2man:B<xcat> - Extreme Cluster/Cloud Administration Toolkit.
xCAT-confluent/xpod2man:Extreme Cluster/Cloud Administration Toolkit (xCAT). xCAT is a scalable distributed compu
ting management
xCAT-server/share/xcat/tools/go-xcat:   xCAT (Extreme Cluster/Cloud Administration Toolkit): <http://xcat.org/>
xCAT-vlan/xpod2man:B<xcat> - Extreme Cluster/Cloud Administration Toolkit.
xCAT-vlan/xpod2man:Extreme Cluster/Cloud Administration Toolkit (xCAT). xCAT is a scalable distributed computing
management
xCAT/debian/control: xCAT is Extreme Cluster/Cloud Administration Toolkit. xCAT offers complete
[gurevich@c910loginx01 xcat-core]$
```

whowutwut commented on xCAT-UI/debian/control:
```diff
@@ -9,5 +9,4 @@ Package: xcat-ui
 Description: Provides a browser-based interface for xCAT
- Provides a browser-based interface for xCAT (extreme Cluster Administration
- Tool).
+ Provides a browser-based interface for xCAT (Extreme Cluster/Cloud Administration Toolkit).
```
What if we remove `(Extreme Cluster/Cloud Administration Toolkit)`

whowutwut commented on xCAT-UI/lib/ui.php:
```diff
@@ -72,7 +72,7 @@ function login() {
             <table>
                 <tr><td colspan=5></td></tr>
-                <tr><td align=right><img src="images/logo.png" width="50" height="35"></img></td><td colspan=4 style="font-size: 18px;">eXtreme Cloud Administration Toolkit</td></tr>
+                <tr><td align=right><img src="images/logo.png" width="50" height="35"></img></td><td colspan=4 style="font-size: 18px;">Extreme Cluster/Cloud Administration Toolkit</td></tr>
```
What about just writing "xCAT"

whowutwut commented on xCAT-UI/service.php:
```diff
@@ -57,7 +57,7 @@
             <table>
                 <tr><td colspan=5></td></tr>
-                <tr><td align=right><img src="images/logo.png" width="50" height="35"></img></td><td colspan=4 style="font-size: 18px;">eXtreme Cloud Administration Toolkit</td></tr>
+                <tr><td align=right><img src="images/logo.png" width="50" height="35"></img></td><td colspan=4 style="font-size: 18px;">Extreme Cluster/Cloud Administration Toolkit</td></tr>
```
What about just writing xCAT

whowutwut commented on xCAT-UI/xCAT-UI.spec:
```diff
@@ -31,7 +31,7 @@ Requires: httpd
 
 %description
-Provides a browser-based interface for xCAT (Extreme Cloud Administration Toolkit).
+Provides a browser-based interface for xCAT (Extreme Cluster/Cloud Administration Toolkit).
```
Let's remove "Extreme Cluster/Cloud Adminstration Toolkit"

whowutwut commented on docs/source/index.rst:
```diff
@@ -1,4 +1,4 @@
-Extreme Cloud/Cluster Administration Toolkit
+Extreme Cluster/Cloud Administration Toolkit
```
What if we changed this heading to just be "xCAT" we have the meaning below.

whowutwut commented on perl-xCAT/README:
```diff
@@ -1,4 +1,4 @@
-xCAT - eXtreme Cloud Administration Toolkit
+xCAT - Extreme Cluster/Cloud Administration Toolkit
```
Do we need to put what xCAT stands for here?

whowutwut commented on xCAT-OpenStack-baremetal/xpod2man:
```diff
@@ -145,11 +145,11 @@ sub writesummarypage {
 =head1 NAME
 
-B<xcat> - extreme Cluster Administration Tool.
+B<xcat> - Extreme Cluster/Cloud Administration Toolkit.
```
Applies to all the xpod2man, where is this text generated and what happens if we just say "xCAT"

gurevichmark commented:
New commit to answer questions and suggestions from the @whowutwut review:

* Removed most references to the full name and replaced with just `xCAT`
* In some places, where I think the full name should be written down, changed the full name to `Extreme Cloud Administration Toolkit` to match the name in IBM ordering system.
* xpod2man is used to generate the `man` and `.html` files for each xCAT command when `xCAT-client` is installed. One of the pages it generates is a summary page, where I think the full name should stay. That page is displayed by `man xcat` and looks something like this:
```
XCAT.1(1)                             User Contributed Perl Documentation                            XCAT.1(1)

NAME
       xcat - extreme Cloud Administration Tool.

DESCRIPTION
       Extreme Cloud Administration Toolkit (xCAT). xCAT is a scalable distributed computing management and
       provisioning tool that provides a unified interface for hardware control, discovery, and OS
       diskful/diskfree deployment.

XCAT DATABASE
       All of the cluster configuration information is in the xCAT database.  See xcatdb(5) for descriptions
       of every table in the database.

XCAT ADDITIONAL TOOLS
       Some additional tools have been contributed to xCAT.  You can read about them at
       http://xcat-docs.readthedocs.org/en/latest/guides/admin-guides/references/index.html or in
       /opt/xcat/share/xcat/tools/README.txt on your xCAT management node.

XCAT COMMANDS
       What follows is a short description of each xCAT command.  To get more information about a particular
       command, see its man page.  Note that the commands are listed in alphabetical order within each
       section, i.e. all the commands in section 1, then the commands in section 3, etc.

       addkit(1)   Adds product software Kits to an xCAT cluster environmnet.

       addkitcomp(1)
                   Assign Kit components to an xCAT osimage.

       bmcdiscover(1)
                   Discover Baseboard Management Controllers (BMCs) using a scan method

       cfgve(1)    Configure the elements for a virtual environment.
:
:
```

gurevichmark commented:
Close/open to restart Travis

whowutwut commented on xCAT-OpenStack-baremetal/xpod2man:
```diff
@@ -22,9 +22,6 @@ my @pods = getPodList($poddir);
 
 #foreach (@pods) { print "$_\n"; } exit;
 
-# Build the cmd overview page.
```
Should we get rid of that # above at line 23 and below at line 29?

whowutwut commented on xCAT-server/share/xcat/tools/go-xcat:
```diff
@@ -63,7 +63,7 @@ function usage()
 	          --xcat-dep=http://xcat.org/path/to/xcat-dep.tar.bz2 install
 
-	xCAT (Extreme Cloud/Cluster Administration Toolkit): <http://xcat.org/>
+	xCAT: <http://xcat.org/>
```
Any reason for that URL to be wrapped in `< >`  ?

gurevichmark commented:
New commit for last review comments
